src/metadata_validator.py

```diff
@@ -314,7 +314,7 @@ def validate_relationship(self, data_record):
         data_record_parent_nodes = data_record.get(PARENTS)
         if not data_record_parent_nodes or len(data_record_parent_nodes) == 0:
             result["result"] = STATUS_WARNING
-            result[ERRORS].append(create_error("Relationship not specified", f'{msg_prefix} No relationships specified.'))
+            result[ERRORS].append(create_error("Relationship not specified", f'{msg_prefix} All relationship columns are missing. Please ensure at least one relationship column is included.'))
```

Comment:
I think this will only happen if all relationship cells are empty, missing column should be blocked at essential validation, if so, let's change this error to "All related node IDs are missing. Please ensure at least one related node ID is included."

Reply:
Updated.